externs/olx.js

```diff
@@ -2025,6 +2026,17 @@ olx.format.GeoJSONOptions.prototype.geometryName;
 
 
 /**
+ * Certain GeoJSON providers include the geometry_name field in the feature
```

Comment:
You should also document that the default is `false`.

Reply:
Thanks for the review. The documentation was updated.